common-design-patterns/most-requested.html

```diff
@@ -1,177 +1,41 @@
+---
+title: Most requested
```

Comment:
We need the `altLangPage` property to link to `gc-proto/design-system-demo/configurations-conception-communes/en-demande.html`

Reply:
And the `dateModified` property

Reply:
And the `breadcrumbs`

Reply:
done